Formula/criterion.rb

```diff
@@ -0,0 +1,38 @@
+class Criterion < Formula
+  desc "Cross-platform C and C++ unit testing framework for the 21st century"
+  homepage "https://github.com/Snaipe/Criterion"
+
+  url "https://github.com/Snaipe/Criterion/releases/download/v2.4.0/criterion-2.4.0.tar.xz"
+  sha256 "b13bdb9e007d4d2e87a13446210630e95e3e3d92bb731951bcea4993464b9911"
+  license "MIT"
+
+  head "https://github.com/Snaipe/Criterion.git", branch: "bleeding"
+
+  depends_on "cmake" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "libffi"
+  depends_on "libgit2"
+
+  def install
+    system "meson", "setup", *std_meson_args, "--wrap-mode=default", "build"
```

Comment:
```suggestion
    system "meson", "setup", *std_meson_args, "build"
```
We set `--wrap-mode` in `std_meson_args`. We shouldn't override that value (or else why include it in `std_meson_args` to begin with).

Reply:
Thanks for the note. I've replaced it with a less intrusive `--force-fallback-for=boxfort` flag.

Boxfort is a dependency that is strongly coupled with Criterion, Criterion seems to be the only known user of this library.
Its current version is v0.1.4, which can be considered as an experimental state when used outside of Criterion, so currently, it might not be worth adding a new Homebrew formula for it.